front/js/cart.js

```diff
@@ -1,15 +1,158 @@
     }
-}
\ No newline at end of file
+
+    displayCart(productList);
+    console.log('liste des produits', productList);
+};
+
+getProductCard();
+
+
+// //Intégration dynamique des éléments dans le DOM
+
+function displayCart(productList) {
+
+    // Pour chaque objets dans l'array liste du panier
+    // Si la couleur et l'id du produit sont les mêmes
+
+
+
+
+    for (let item in productList) {
+
+        const product = productList[item];
+        const idProduct = productList[item]._id;
+
+        // Insertion dynamique des éléments
+
+        let cartItemArticle = document.createElement('article');
+        document.getElementById('cart__items').appendChild(cartItemArticle);
+        cartItemArticle.className = 'cart__item';
+        cartItemArticle.setAttribute('data-id', idProduct);
+        cartItemArticle.setAttribute('data-color', localStorageArea[item].colorChoice);
+
+        let cartItemImgDiv = document.createElement('div');
+        cartItemArticle.appendChild(cartItemImgDiv);
+        cartItemImgDiv.className = 'cart__item__img';
+
+        let imgItem = document.createElement('img')
+        cartItemImgDiv.appendChild(imgItem);
+        imgItem.src = product.imageUrl;
+        imgItem.alt = product.altTxt;
+
+        let cartItemContentDiv = document.createElement('div');
+        cartItemImgDiv.appendChild(cartItemContentDiv);
+        cartItemContentDiv.className = 'cart__item__content';
+
+        let cartItemContentDescriptionDiv = document.createElement('div');
+        cartItemContentDiv.appendChild(cartItemContentDescriptionDiv);
+        cartItemContentDescriptionDiv.className = 'cart__item__content__description';
+
+        let productName = document.createElement('h2');
+        cartItemContentDescriptionDiv.appendChild(productName);
+        productName.textContent = product.name;
+
+        let productColor = document.createElement('p');
+        cartItemContentDescriptionDiv.appendChild(productColor);
+        productColor.textContent = localStorageArea[item].colorChoice;
+
+        let productPrice = document.createElement('p');
+        cartItemContentDescriptionDiv.appendChild(productPrice);
+        productPrice.textContent = product.price + ' €';
+
+        let cardItemContentSettingsDiv = document.createElement('div');
+        cardItemContentSettingsDiv.className = 'cart__item__content__settings';
+        cartItemContentDiv.appendChild(cardItemContentSettingsDiv);
+
+        let cartItemContentSettingsQuantityDiv = document.createElement('div');
+        cartItemContentSettingsQuantityDiv.className = 'cart__item__content__settings__quantity';
+        cardItemContentSettingsDiv.appendChild(cartItemContentSettingsQuantityDiv);
+
+        let quantityProduct = document.createElement('p');
+        cartItemContentSettingsQuantityDiv.appendChild(quantityProduct);
+        quantityProduct.textContent = 'Qté : ';
+
+        let InputModifyProduct = document.createElement('input');
```

Comment:
```suggestion
        let inputModifyProduct = document.createElement('input');
```
Histoire d'avoir la même règle de nommage partout

Reply:
Bien vu ^^ merci